src/tests/unit/dhcpv6_rfc3319.txt

```diff
@@ -0,0 +1,36 @@
+#
+#  Test vectors for DHCPv6 attributes for (SIP) Servers
+#
+load dhcpv6
+dictionary-load dhcpv6
+
+#
+#  3.1  SIP Servers Domain Name List
+#
+
+#  domain = mysip.com
+encode-pair SIP-Server-Domain-Name-List = mysip.com
+data 00 15 00 09 6d 79 73 69 70 2e 63 6f 6d
+
+#  ipv6 = 2001:0db8:85a3:0000:0000:8a2e:0370:1234
+encode-pair SIP-Server-Domain-Name-List = 0x2001db885a38a2e3701234
+data 00 15 00 0b 20 01 db 88 5a 38 a2 e3 70 12 34
```

Comment:
That's not the correct output.  DNS attributes should be encoded in DNS wire format as per the RFC

Reply:
Don't assume the encoder is correct.  The point is to come up with tests that actually check the output is right.  There's lots of places where it's not complete.  I got through writing the encoder and then realised we needed major work on other areas of the server like splitting the protocols out into their own dictionaries, and being able to create grouping attributes.